README.md

```diff
@@ -4,8 +4,9 @@ A package of resources for Hypothesis front-end applications.
 
 #### Requirements
 
+Dependencies **required** for importing modules from `@hypothesis/frontend-shared`:
+
 - preact
```

Comment:
browserify is not really a requirement at all so I think it's best not to confuse this with a actual  dependency such as preact.

Reply:
I would probably suggest rewording this section, including the list, as:

"Your project must have Preact installed as a dependency."

The main change here is just to make it clear that this is an unconditional dependency. npm/yarn will expect Preact to be installed whether or not you actually use any UI components, as long as preact is listed as a mandatory peer dependency. 

In writing the above comment I tried to find a link in the Preact docs that would tell you how to install Preact with yarn and npm but they miss that step, so perhaps we could add it here (`yarn install --save preact`).